### ISSUE TITLE
awscli 2.0 support

### ISSUE THREAD

**Issue by bensternlieb** [CircleCI-Public/aws-eks-orb#29]
## What would you like to be added

Folks, I'm seeing an install of awscli 1.x after explicitly installing awscli 2.0.  Can a check be done first for existing awscli install?
```
orbs:
  aws-cli: circleci/aws-cli@1.0.0
  aws-eks: circleci/aws-eks@0.2.7

jobs:
  test:
    executor: aws-eks/python3
    steps:
      - checkout
      - aws-cli/setup
      - aws-eks/update-kubeconfig-with-authenticator:
          cluster-name: dl-k8s
          install-kubectl: true
```

## Why is this needed
Overhead of additional aws install is large and unnecessary.

**Comment by danburkert** [comment]
This is a really painful issue for my organization, where this crate is causing >45s of wasted CI time per job unnecessarily installing aws-cli via pip. I've opened #30 to try and get the ball rolling here.

**Comment by lokst** [comment]
This should now be resolved by #30 . Please try using version `1.0.0` of the orb

**Comment by ianhundere** [comment]
i'm experiencing issues where env vars are not being read from $BASH_ENV
